honeybee_schema/radiance/properties.py

```diff
@@ -0,0 +1,173 @@
+
+from .._base import IDdBaseModel
+from .modifier import Void, ModifierBase
+from .modifierset import ModifierSet
+
+
+class _PropertiesBaseAbridged(IDdBaseModel):
+    """Base class of Abridged Radiance Properties."""
+
+    modifier: str = Field(
+        default=None,
+        min_length=1,
+        max_length=100,
+        description='A string for a Honeybee Radiance Modifier.'
+        )
+
+    modifier_blk: str = Field(
+        default=None,
+        min_length=1,
+        max_length=100,
+        description='A string for a Honeybee Radiance Modifier to be used '
+                    'in direct solar simulations and in isolation studies (assessing'
+                    'the contribution of individual objects).'
+        )
+
+
+class _PropertiesBase(IDdBaseModel):
+    """Base class of Radiance Properties."""
+
+    modifier: ModifierBase = Field(
+        default=None,
+        description='A Honeybee Radiance Modifier for the object.'
+        )
+
+    modifier_blk: ModifierBase = Field(
+        default=None,
+        description='A Honeybee Radiance Modifier to be used for this object'
+                    'in direct solar simulations and in isolation studies (assessing'
+                    'the contribution of individual objects).'
+        )
+
+
+class ApertureRadiancePropertiesAbridged(_PropertiesBase):
+    """Radiance Properties for Honeybee Aperture Abridged."""
+
+    type: constr(regex='^ApertureRadiancePropertiesAbridged$') = \
+        'ApertureRadiancePropertiesAbridged'
+
+
+class ApertureRadianceProperties(_PropertiesBase):
+    """Radiance Properties for Honeybee Aperture."""
+
+    type: constr(regex='^ApertureRadianceProperties$') = 'ApertureRadianceProperties'
+
+
+class DoorRadiancePropertiesAbridged(_PropertiesBaseAbridged):
+    """Radiance Properties for Honeybee Door Abridged."""
+
+    type: constr(regex='^DoorRadiancePropertiesAbridged$') = \
+        'DoorRadiancePropertiesAbridged'
+
+
+class DoorRadianceProperties(_PropertiesBase):
+    """Radiance Properties for Honeybee Door."""
+
+    type: constr(regex='^DoorRadianceProperties$') = 'DoorRadianceProperties'
+
+
+class FaceRadiancePropertiesAbridged(_PropertiesBaseAbridged):
+    """Radiance Properties for Honeybee Face Abridged."""
+
+    type: constr(regex='^FaceRadiancePropertiesAbridged$') = \
+        'FaceRadiancePropertiesAbridged'
+
+
+class FaceRadianceProperties(_PropertiesBase):
+    """Radiance Properties for Honeybee Face."""
+
+    type: constr(regex='^FaceRadianceProperties$') = 'FaceRadianceProperties'
+
+
+class ShadeRadiancePropertiesAbridged(_PropertiesBaseAbridged):
+    """Radiance Properties for Honeybee Shade Abridged."""
+
+    type: constr(regex='^ShadeRadiancePropertiesAbridged$') = \
+        'ShadeRadiancePropertiesAbridged'
+
+
+class ShadeRadianceProperties(_PropertiesBase):
+    """Radiance Properties for Honeybee Shade."""
+
+    type: constr(regex='^ShadeRadianceProperties$') = 'ShadeRadianceProperties'
+
+
```

Comment:
Where are the `RoomRadiancePropertiesAbridged`? These should have properties like what you see in the sample here:
https://github.com/ladybug-tools/honeybee-schema/blob/master/samples/model/model_complete_multiroom_radiance.json#L1652-L1655

Reply:
Added, I forgot to add this.